### ISSUE TITLE
Clear cache button text unclear

### ISSUE THREAD

**Issue by ViRb3** [marcowindt/WiiMoteDSU#6]
The button in the setting is currently called "clear cache". This is technically correct, but the naming hides the fact that clicking on the button will actually reset the orientation of the gyroscope, effectively calibrating it. In my opinion, it would be clearer to name it "Calibrate sensor" or "Reset sensor" or something similar.

**Comment by marcowindt** [comment]
I agree the button text should be something different. I changed it to "Reset to default" in d4444ae37916bcea976c590165d640b628f64c1a, since the main idea is to reset the settings to the default values.

**Comment by ViRb3** [comment]
That's perfect, thank you!

**Comment by ViRb3** [comment]
Actually, does the button still reset the origin (perform calibration)? If yes, I feel like "Reset to default" still doesn't make that clear enough. Maybe have two buttons, one to reset the settings, and one to reset the origin (calibrate)?

**Comment by marcowindt** [comment]
No, it only resets whether it should invert the values (or enable/disable the accelerometer input) from the sensors that gets send to dolphin.

**Comment by ViRb3** [comment]
Very strange. I could have sworn that last time I tested v1.0.0, when I was at Dolphin Controller Settings > Wii Remote Configure > Motion Input, clicking "Reset cache" in the app would reset the origin (blue point) to the middle, effectively centering the phone with the way I'm holding the phone. I tried this again today, both with v1.0.1 and v1.0.0, and I can't reproduce it. I updated Dolphin a few days ago, so I suppose it could be that?

Do you have a Discord or Telegram where I could contact you and ask a few more questions?

Thanks

**Comment by marcowindt** [comment]
From my understanding this is Dolphin doing the calibration, if you put the device on a flat surface (or hold still).

![dolphin-gryo-calibration](https://user-images.githubusercontent.com/9336858/110187475-c4bd7e80-7e18-11eb-97f9-bfbef89bcd0d.gif)
